### DISCUSSION TITLE
Help in getting started in open source

### DISCUSSION THREAD

**Discussion by i-dipanshu** [cncf/mentoring#714] [Q&A] [answered]
Hello Everyone, 
I'm a 1st year undergrad and new to open source community.
I have a good understanding of some devops tech stacks and wanted to contribute and help with what i can.
So, i digged into github repo of Big projects like kubernetes to understand their codebase.
This would also help in getting into GSoC 2023,
but i got confused , what issues to choose and get started
Any kind of help would be highly appreciated

**Comment by thisisobate** [comment, marked as answer]
Hi @i-dipanshu,
First off, welcome to the community! 

Since you're generally new to open source, I suggest you start with **good first issues**. Tackling good first issues is a great way to familiarize yourself with a new codebase. It's usually designed to be easy allowing newcomers to **"learn by doing"**. 

To find a good first issue, navigate to the project's GitHub repository. Click the "issues" tab and filter using the label "good first issue". 

Hope this helps!

**Comment by satyampsoni** [comment]
Hi @i-dipanshu 
Glad you have learnt  these much in your first year, I just wanted to say  while getting started you don't need to understand the whole codebase. All way round it will confuse you and make it overwhelming for you. start with good first issues and keep contributing.

**Reply by i-dipanshu** [reply]
Thank you @satyampsoni for your advice.
It has been a while since I asked this 
Happy to share that I am really enjoying doing open source
